<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>265</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(223, 222, 221);
	border-radius:15px;
	color: rgb(0, 0, 0);
	
	font: 300 10pt &quot;Ubuntu Light&quot;;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="portCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Baud Rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baudrateCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Data Bits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="databitsCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Stop Bits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopbitsCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parityCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Flow Control</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="flowcontrolCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>No Flow Control</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeCombo">
          <property name="styleSheet">
           <string notr="true">font: 300 9pt &quot;Ubuntu Light&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>Anomaly Detection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>n-Classification</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.042, y1:0.0397727, x2:0.949, y2:0.971591, stop:0 rgba(4, 170, 145, 255), stop:0.812207 rgba(1, 21, 18, 255));
color: rgb(255, 255, 255);
border-radius: 8px;s</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.042, y1:0.0397727, x2:0.949, y2:0.971591, stop:0 rgba(4, 170, 145, 255), stop:0.812207 rgba(1, 21, 18, 255));
color: rgb(255, 255, 255);
border-radius: 8px;s</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
